integration/emr/alluxio-presto.json

```diff
@@ -0,0 +1,40 @@
+[
+  {     
+    "Classification": "hive-site",
+    "Properties": {
+      "fs.alluxio.impl": "alluxio.hadoop.FileSystem",
```

Comment:
is "fs.alluxio.impl" and "fs.AbstractFileSystem.alluxio.impl" required here?

Reply:
Just tested. not required.